### ISSUE TITLE
Unusual behavior when this plugin is introduced... 

### ISSUE THREAD

**Issue by zipzit** [hdhauk/UnrealZeroMQ#3]
I don't understand why, but when I add this folder to my game plugins folder, then restart the game, it absolutely refuses to launch or self -rebuild.  When I attempt to rebuild from Visual Studio, its a fail.    I'm unable to rebuild the base program until I remove this from the plugin folder within my game.  

I'm not complaining, just a warning to others.  I think I'll attempt to rebuild this 3rd party plug in starting with the sample provided in the game engine and utilizing ZeroMQ version libzmq-mt-4_3_3.dll and .pdb and see what happens.  

Many thanks to the author of this plugin for giving me a hint as to how to proceed..

**Comment by getarobo** [comment]
@zipzit hey just wondering if you managed to work zmq in unreal~! Thanks!

**Comment by zipzit** [comment]
Yeah, I did.  And I had a heck of a time getting there.  I started writing up the experience and what I learned... not quite finished yet.  I'll post the link here in a day or so.  Its a pretty long article.  I made every mistake in the book, in learning how to do this right.  And yeah, we should probably generate a pull request on this repo for the update.

**Comment by getarobo** [comment]
Would really appreciate such article, let me know if I can help on somethings~

**Comment by Dene33** [comment]
> Yeah, I did. And I had a heck of a time getting there. I started writing up the experience and what I learned... not quite finished yet. I'll post the link here in a day or so. Its a pretty long article. I made every mistake in the book, in learning how to do this right. And yeah, we should probably generate a pull request on this repo for the update.

Any updates on that? I can help with something related to this repo too.
